specs/phase0/p2p-interface.md

```diff
@@ -387,6 +387,7 @@ The following validations MUST pass before forwarding the `attestation` on the s
   i.e. `compute_subnet_for_attestation(committees_per_slot, attestation.data.slot, attestation.data.index) == subnet_id`,
   where `committees_per_slot = get_committee_count_per_slot(state, attestation.data.target.epoch)`,
   which may be pre-computed along with the committee information for the signature check.
+- _[REJECT]_ The committee index is within the expected range -- i.e. `data.index < get_committee_count_per_slot(state, data.target.epoch)`
```

Comment:
This should probably go before the `== subnet_id` condition. This is kind of a precursor condition to that one